src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table (name = "GROUP")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class Group {
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "GROUP_ID", unique = true)
+    private Long groupId;
+
+    @Column(name = "NAME")
+    @NotNull
+    private String name;
+
+    @OneToMany(targetEntity = Product.class,
+            mappedBy = "group",
+            cascade = CascadeType.ALL,
+            fetch = FetchType.LAZY)
```

Comment:
I asked Witold about it and I'll ask you too: Are you sure this option of cascade will be okej? I don't feel confident with cascading, it can cause many problems that's why I'm asking about it.

Reply:
Deleted :)